topics/variant-analysis/tutorials/trio-analysis/tutorial.md

```diff
@@ -0,0 +1,441 @@
+- Pre-process VCFs using regular expressions and AWK.
+- Use annotations and phenotype information to find the causative variant(s).
+time_estimation: 2H
+key_points:
+- Downloading whole datasets with HTSGET is safe and easy with galaxy.
+- AWK and regex are usefull tools for pre-processing VCF files.
+- Variant annotations allows us to strictly filter VCFs to find the causative variant.
+contributors:
+- Jasper Ouwerkerk
+
+---
+
+# Introduction
+{:.no_toc}
+To discover causal mutations of inherited diseases it’s common practice to do a trio analysis. In a trio analysis DNA is sequenced of both the patient and parents. Using this method, it’s possible to identify multiple inheritance patterns. Some examples of these patterns are autosomal recessive, autosomal dominant, and  de-novo variants, which are represented in the figure below. To elaborate, the most left tree shows an autosomal dominant inhertitance pattern where the offspring inherits a faulty copy of the gene from one of the parents. The center subfigure represents an autosomal recessive disease, here the offspring inherited a faulty copy of the same gene from both parents. In the right subfigure a de-novo mutation is shown, which is caused by a mutation during the offspring’s lifetime.
+
+| ![plot](../../images/trio-analysis/pedigree.svg) |
+|:--:|
+| **Figure 1:** *3 family trees representing autosomal dominant, autosomal recessive, and a de-novo inheritence patter from parents to son.* |
+
+To discover these mutations either whole exome sequencing (WES) or whole genome sequencing (WGS) can be used. With these technologies it is possible to uncover the DNA of the parents and offspring to find (shared) mutations in the DNA. These mutations can include insertions/deletions (indels), loss of heterozygosity (LOH), single nucleotide variants (SNVs), copy number variations (CNVs), and fusion genes. 
+
+In this tutorial we will also make use of the HTSGET protocol, which is a program to download our data securely and savely. This protocol has been implemented in the {% tool [EGA Download Client](toolshed.g2.bx.psu.edu/repos/iuc/ega_download_client/pyega3/4.0.0+galaxy0) %} tool, so we don't have to leave galaxy to retrieve our data.
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+> 
+> 1. 
+> {:toc}
+>
+{: .agenda}
+
+
+# Data preperation
+In this tutorial we will use case 5 from the RD-Connect GPAP synthetic datasets. This data was built using the same three public genomic datasets from HapMap samples NA12877, NA12878, and NA12882, click [here](https://www.genome.gov/10001688/international-hapmap-project) for more info. In these datasets real causative variants which correlate with the phenotypic data are manually spiked-in. 
+
+The dataset consists of WGS from a family trio, in which both the mother and daughter are affected by breast cancer. Our goal is to identify the genetic variation that is responsible for the disease. 

```

Comment:
Please improve this explanation. From what I understand, this is real WGS data from real family trio, but the mutation people are supposed to discover has been spiked in? So the phenotype within that trio was *not* breast cancer?
Please clarify.

Reply:
Exactly! I've made it more clear in: 10ff5199016c7a0b7e15ad5fdf62c69b35666a7f.